examples/DateRangePickerWrapper.jsx

```diff
@@ -73,43 +74,40 @@ const defaultProps = {
   // internationalization
   displayFormat: () => moment.localeData().longDateFormat('L'),
   monthFormat: 'MMMM YYYY',
-  phrases: {
-    closeDatePicker: 'Close',
-    clearDates: 'Clear Dates',
-  },
+  phrases: DateRangePickerPhrases,
 };
 
 class DateRangePickerWrapper extends React.Component {
   constructor(props) {
     super(props);
 
-    let focusedInput = null;
+    let selectedInput = null;
```

Comment:
Renaming from `focused` to `selected` creates a pretty big diff that might make merging/rebasing other PRs hairy. If `selected` is required in a11y props, can the renaming be limited to external facing html props?